src/Compilers/CSharp/Portable/Lowering/Instrumentation/DynamicAnalysisInjector.cs

```diff
@@ -129,6 +129,31 @@ private static bool IsExcludedFromCodeCoverage(MethodSymbol method)
             return false;
         }
 
+        private BoundExpression GetFileIndices()
+        {
+            var documents = PooledHashSet<DebugSourceDocument>.GetInstance();
```

Comment:
Unfortunately, we don't have access to the actual integers in this location in the code (we have access to BoundExpressions for the integers and the DebugSourceDocuments corresponding to the documents and I can only use the latter because the former cannot be compared).